test/k8s-e2e/e2e-test-execute

```diff
@@ -316,7 +316,10 @@ function run_e2e_tests_kubetest() {
 		return $success
 	fi
 
-	isKubeAPIReachable
+	if ! isKubeAPIReachable; then
+		success="$FALSE"
+		return $success
```

Comment:
return immediately without assigning the success var?

Reply:
also success ist already set by the `isKubeAPIReachable` function (https://github.com/gardener/test-infra/blob/master/test/k8s-e2e/e2e-test-execute#L248)